intro-Linux/ch_uploading_files.tex

```diff
@@ -1,26 +1,34 @@
 \chapter{Uploading/downloading/editing files}
 
-\subsection{Uploading/downloading files}
+\section{Uploading/downloading files}
+\label{sec:uploading-files}
 
-Detailed information about uploading/downloading files with ``scp'' (using an OS
-X or Linux client) or WinSCP (Windows client), see
-\url{http://hpc.ugent.be/userwiki/index.php/User:VscCopy}.
+To transfer files from and to the HPC, see
+\href{\HPCManualURL#sec:filetranfer}{the section about tranferring files in chapter 3 of the HPC manual}.
+
+\ifwindows
 
 After copying files it is advised to run
+\hypertarget{sec:dos2unix}{}
```

Comment:
Why not add a `subsubsection` here, it'll make it stands out a bit more, and you can then more easily refer to it?